rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+
+
+### Tier II
+
+#### Armor Resizing Kit
+> This kit has everything a blacksmith needs to adjust suits of armor to better suit their needs. A Smith using this kit may increase an existing suit of armor's value rating by spending additional resources. This does not change the durability of the armor. This has 5 charges.
+#### Lockpicks
+   >This is a basic item for use with the Lockpicking skill. Having lockpicks lowers the rating of the lock by 1. Lockpicks have 10 charges.
+#### Suit of Armor
+   >This creates a finely made suit of armor that has an Armor rating of 10, with 5 Durability. You can increase the Tier of the recipe and the cost in materials by 1 to increase the Armor Rating by 10, up to a max of 80.
+
+
+
+### Tier III
+
+#### Hardening
+   >This will harden a weapon, shield, armor or magic item to Resist Destruction once. An item may only be Hardened up to three times (For a maximum of Three Resist Destructions) May be applied to untagged items. (Such as Doors).
+#### Knight-Night Helmet
+   >This specially designed helmet for knights who have too often fell victim to physically-induced-narcolepsy. This helmet will let the wearer Minimize any Waylay effect they receive. This has 5 charges, and must be repped.
+#### Repair Lock
+   >The smith is able to fix a previously broken lock. The repair of the lock takes 1 minute of  time in addition to the three ranks of raw materials. This is not a tagged item.
+
+### Tier IV
+
+#### Lock Hardening
+   >This recipe will harden a lock so that the lock will resist destroy effects, and be more difficult to pick. Hardening a lock will allow the lock to resist five destroy effects, and will raise the lockpicking rating of the lock by three. Hardening may only be done once on an item.
+#### One-Handed Weapon
+   >This creates a finely made One-Handed Weapon that has a base damage of 1, with 5 Durability.
+#### Shield
+   >This creates a finely made Shield, with 5 Durability.
+
+### Tier V
+
+#### Silver Weapon Forging
+   >With this skill the blacksmith can carefully silver plate a weapon in order for the weapon to do to the silver damage type. This can be done at time of creation or to an already existing weapon.
+#### Two-Handed Weapon
+   >This creates a finely made Two-Handed Weapon that has a base damage of 1, with 5 Durability.
+#### Whetstone 
+   >A user can spend one minute of careful sharpening using this tool to improve a single weapon's base damage by 1 for 5 days. This process however shorten's the item's lifespan, reducing it's durability by 1. If this would reduce the durability to 0, the item is destroyed. This item cannot stack with similar effects. This item does not work on Unarmed or Claws. This item is consumed on use.
```

Comment:
Can we update `one minute` to `1 minute` to match conventions?

`reducing it's` - it's should be its.